eZ/Publish/API/Repository/SearchService.php

```diff
@@ -103,13 +105,11 @@ interface SearchService
      * @since 6.12 (constant added in 6.7.6 and up)
      * @see \eZ\Publish\API\Repository\Values\Content\Query\Criterion\FullText
      */
-    const CAPABILITY_ADVANCED_FULLTEXT = 64;
+    public const CAPABILITY_ADVANCED_FULLTEXT = 64;
 
     /**
      * Finds content objects for the given query.
      *
-     * @throws \eZ\Publish\API\Repository\Exceptions\InvalidArgumentException if query is not valid
```

Comment:
Why we remove it? In the implementation it is still present https://github.com/ezsystems/ezpublish-kernel/pull/2852/files#diff-d25d842417f878e3583966990cccaae0R85

Reply:
Fixed in https://github.com/ezsystems/ezpublish-kernel/pull/2852/commits/7c06c7d64860dbb20a0a55f4d5d028c5d3b9368a